WebDriverAgentTests/IntegrationTests/FBImageIOScalerTests.m

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <XCTest/XCTest.h>
+#import "FBImageIOScaler.h"
+
+@interface FBImageIOScalerTests : XCTestCase
```

Comment:
Please inherit from FBIntegrationTestCase. It already contains all the presetup stuff